<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDiag</class>
 <widget class="QDialog" name="InputDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Animal</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout">
    <item>
     <widget class="QTabWidget" name="infoTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="basic">
       <attribute name="title">
        <string>Basic Info</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>411</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="BasicInfo">
         <item row="13" column="1">
          <widget class="QTextEdit" name="bioTextEdit"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="ageLabel">
           <property name="text">
            <string>               Age:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="sexLabel">
           <property name="text">
            <string>               Sex:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="sexSelector">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Sex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="12" column="1">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="breedLineEdit"/>
         </item>
         <item row="11" column="0" colspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="bioLabel">
           <property name="text">
            <string>Biography</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>          Name:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="ageSpinBox">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSlider" name="careSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="careLabel">
           <property name="text">
            <string>Level of Care: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="speciesSelector">
           <item>
            <property name="text">
             <string>Species</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dog</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="breedLabel">
           <property name="text">
            <string>           Breed:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="speciesLabel">
           <property name="text">
            <string>        Species:</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="social">
       <attribute name="title">
        <string>Social</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>411</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="SocialInfo">
         <item row="4" column="0">
          <widget class="QLabel" name="animalLabel">
           <property name="text">
            <string>   Affinity, Animals: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="childLabel">
           <property name="text">
            <string>   Affinity, Children: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="peopleLabel">
           <property name="text">
            <string>     Affinity, People: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="trainingLabel">
           <property name="text">
            <string>                Training: </string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="timeLabel">
           <property name="text">
            <string>Time Commitment: </string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="approachabilityLabel">
           <property name="text">
            <string>    Approachability: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="trainingSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="peopleSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="childSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="animalSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="approachabilitySlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSlider" name="timeSlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="trainabilityLabel">
           <property name="text">
            <string>           Trainability:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="trainabilitySlider">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="sliderPosition">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="history">
       <attribute name="title">
        <string>History</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>411</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="2">
          <widget class="QLineEdit" name="mobilityLineEdit">
           <property name="placeholderText">
            <string>Mobility Restrictions</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="aHistLabel">
           <property name="text">
            <string>      History of Abuse: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string> Dietary Restrictions: </string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QTextEdit" name="aHistTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="immunizedLayout">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="immunizedCheckbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="immunizedLabel">
           <property name="text">
            <string>              Immunized: </string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>               Disabilities: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>50</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>         Mobility Needs: </string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="disabLineEdit">
           <property name="placeholderText">
            <string>Physical Disabilities</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="dietLineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="placeholderText">
            <string>Dietary Restrictions</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="buttons" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>198</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="saveButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancelButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>speciesSelector</tabstop>
  <tabstop>breedLineEdit</tabstop>
  <tabstop>sexSelector</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>careSlider</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>infoTabWidget</tabstop>
  <tabstop>trainingSlider</tabstop>
  <tabstop>peopleSlider</tabstop>
  <tabstop>childSlider</tabstop>
  <tabstop>animalSlider</tabstop>
  <tabstop>approachabilitySlider</tabstop>
  <tabstop>timeSlider</tabstop>
  <tabstop>immunizedCheckbox</tabstop>
  <tabstop>aHistTextEdit</tabstop>
  <tabstop>bioTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
